<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDBGeo</class>
 <widget class="QDialog" name="CDBGeo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>98</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>98</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Geocoder</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>10</y>
     <width>441</width>
     <height>111</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="lineEditPlaceName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGeocode">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QComboBox" name="cbxGeocodeMatches">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOk">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
